packages/guilded.js/lib/structures/managers/MessageManager.ts

```diff
@@ -24,6 +25,32 @@ export class MessageManager extends BaseManager<APIMessage | UpgradedMessageData
         return this.client.channels.deleteMessage(this.channel.id, msg);
     }
 
+    /**
+     * Add a reaction to this message (UNFINISHED)
+     * @hidden
+     */
+    public react(emoji: string, channel: string | PartialChannel, message: string | Message): unknown {
```

Comment:
What do you think about ordering the function params like `message, channel, emoji`?